analyzer/src/funTest/assets/projects/synthetic/cargo-subcrate/Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "lib"
 version = "0.1.0"
-authors = ["anon <anon@example.org>"]
```

Comment:
Why was this removed? To test the implementation if *no* author is given?

Reply:
Yes, exactly for this reason.